packages/components/src/Menu/MenuItem.tsx

```diff
@@ -167,22 +173,34 @@ const MenuItemInternal: FC<MenuItemProps> = (props) => {
     >
       <Component href={href} role="menuitem" target={target}>
         {renderedIcon}
-        <span>{children}</span>
+        <span>
+          {children}
+          {description && (
+            <Paragraph fontSize="xsmall" color="text2">
```

Comment:
I think you can just put the margin here:
```suggestion
            <Paragraph fontSize="xsmall" mt="xxsmall" color="text2">
```